bowling_challenge.js

```diff
@@ -0,0 +1,74 @@
+class Game
+{
+  // this is the constructor for the Game class
+  constructor()
+  {
+    // this is the array of rolls
+    this.rolls = [];
+  }
+  roll(pins)
+  {
+    // this is the method for rolling the ball
+    // it works by pushing the number of pins knocked down into the rolls array
+    this.rolls.push(pins);
+  }
+  score()
+  {
+    let score = 0;
```

Comment:
Well done for adding so many comments. It makes it a lot easier to understand it!